### DISCUSSION TITLE
Pazara elma satmaya giden arkadaşlar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#61] [Zeka Sorusu]
Pazara elma satmaya giden iki arkadaş yolda bir sepet içerisinde 20 elma bulurlar. Bu 20 elmaya sahip olmak için şöyle bir iddiaya tutuşurlar.
İkiside birer çuvala kendi elmalarından koyup daha sonra karşılaştıracaklardır.
Kimin çuvalından daha çok elma çıkarsa iddiayı o kazanaktır.
Eşitlik durumunda ise elmalar paylaşılacaktır.
Yaptıkları anlaşmaya göre, iddiayı kazanan sepeti, kaybeden ise çuvalları alacaktır.
Siz olsaydınız, çuvalınıza kaç elma koyardınız?

**Comment by HabilCabbar** [comment]
(Hocam ben bu arada HabilCcabbar ile aynı kişiyim sadece belirli teknik sorunlardan dolayı bu hesaptayım.)
*9 veya 10 elma**
Her iki durumda da en kötü olasılıkla 10 elma kazanırım.
